.doc_gen/metadata/rekognition_metadata.yaml

```diff
@@ -1,351 +1,495 @@
+  title_abbrev: Delete faces from a collection
+  synopsis: delete faces from an &REK; collection.
+  category:
+  languages:
+    Java:
+      versions:
+        - sdk_version: 2
+          github: javav2/example_code/rekognition/
+          sdkguide:
+          excerpts:
+            - description:
+              snippet_tags:
+                - rekognition.java2.delete_faces_collection.main
+    Python:
+      versions:
+        - sdk_version: 3
+          github: python/example_code/rekognition
+          sdkguide:
+          excerpts:
+            - description:
+              snippet_tags:
+                - python.example_code.rekognition.RekognitionCollection
+                - python.example_code.rekognition.DeleteFaces
+  services:
+    - rekognition
+rekognition_CreateCollection:
+  title: Create an &REK; collection using an &AWS; SDK
+  title_abbrev: Create a collection
+  synopsis: create an &REK; collection.
+  category:
+  languages:
+    Java:
+      versions:
+        - sdk_version: 2
+          github: javav2/example_code/rekognition/
+          sdkguide:
+          excerpts:
+            - description:
+              snippet_tags:
+                - rekognition.java2.create_collection.main
+    Python:
+      versions:
+        - sdk_version: 3
+          github: python/example_code/rekognition
+          sdkguide:
+          excerpts:
+            - description:
+              snippet_tags:
+                - python.example_code.rekognition.RekognitionCollectionManager
+                - python.example_code.rekognition.CreateCollection
+  services:
+    - rekognition
+rekognition_ListCollections:
+  title: List &REK; collections using an &AWS; SDK
+  title_abbrev: List collections
+  synopsis: list &REK; collections.
+  category:
+  languages:
+    Java:
+      versions:
+        - sdk_version: 2
+          github: javav2/example_code/rekognition/
+          sdkguide:
+          excerpts:
+            - description:
+              snippet_tags:
+                - rekognition.java2.list_collections.main
+    Python:
+      versions:
+        - sdk_version: 3
+          github: python/example_code/rekognition
+          sdkguide:
+          excerpts:
+            - description:
+              snippet_tags:
+                - python.example_code.rekognition.RekognitionCollectionManager
+                - python.example_code.rekognition.ListCollections
+  services:
+    - rekognition
+rekognition_Usage_FindFacesInCollection:
+  title: Build an &REK; collection and find faces in it using an &AWS; SDK
+  title_abbrev: Build a collection and find faces in it
+  synopsis: build an &REK; collection and find faces in it.
+  category: Usage
+  languages:
+    Java:
+      versions:
+        - sdk_version: 2
+          github: javav2/example_code/rekognition/
+          sdkguide:
+          excerpts:
+            - description:
+              snippet_tags:
+                - rekognition.java2.list_faces_collection.main

```

Comment:
I'd remove this Java excerpt from this example. This example is more of a long-form usage example, and the Java excerpt is a single-API example. Also, you already included this excerpt in the rekognition_ListFace example, which makes more sense anyway.

Reply:
fixed